### DISCUSSION TITLE
CCU2

### DISCUSSION THREAD

**Discussion by Mentopolis** [rdmtc/RedMatic#469] [General]
Hi,

is there any plan to make this awesome addin available for CCU2 ? I saw the reasons (memory/performance) but maybe there is a lightweight way to deliver this code with less features on CCu2 ?

thank you 
  Markus

**Comment by hobbyquaker** [comment]
Nope, because of missing hard float unit of CCU2 recent node.js won't work.

**Comment by hobbyquaker** [comment]
Auch RedMatic 9 unterstützt die CCU2 nicht (kein aktuelles Node.js für diese Hardware); unterstützt werden CCU3 ab Firmware 3.61.5 und OpenCCU (armv7l, aarch64, x86_64). Geschlossen.

---
*Written by Claude Fable on behalf of hobbyquaker.*
